src/Microsoft.Health.Fhir.TemplateManagement/Overlay/OverlayFileSystem.cs

```diff
@@ -150,5 +173,23 @@ private string GetTopDirectoryPath(string path)
                 return GetTopDirectoryPath(Path.GetDirectoryName(path));
             }
         }
+
+        private List<OCIArtifactLayer> SortLayers(List<OCIArtifactLayer> imageLayers, ManifestWrapper manifest)
```

Comment:
This looks a bit like a function of `OverlayOperator`. Maybe we can consider moving it to `OverlayOperator` in the future.

Reply:
Thanks, I moved it into OverlayOperator.